sprint11/newTasks.js

```diff
@@ -0,0 +1,43 @@
+/*
+  Zadanie 1. Stwórz globalną tablię liczb w tym pliku o nazwie 'numbers'. Dodaj tam liczby:
+  [45, 56, 1, -4, 6, 7, 8, 9]
+  Zadanie 1. Stwórz funkcję o nazwie 'findNumberIndex()'. Niech funkcja przyjmie jeden parametr o nazwie 'number'.
+  Jeżeli 'number' istnieje w tablicy 'numbers', niech funkcja zwróci jej indeks.
+  Jeżeli 'number' nie istnieje w tablicy, niecj funkcja zwróci -1.
+
+  Np.
+  findNumberIndex(6) powinna zwrócić 4
+  findNumberIndex(9) powinna zwrócić 7
+  findNumberIndex(45) powinna zwrócić 0
+  findNumberIndex(999) powinna zwrócić -1
+  findNumberIndex(99) powinna zwrócić -1
+
+  Zadanie 2. Stwórz funkcję, o nazwie 'removeElementFromArray()'. Niech funkcja przyjmie jeden parametr o nazwie 'elementToRemove',
+  Użyj funkcji 'findNumberIndex()', żeby usunąć element z tablicy 'numbers'.
+*/
+
+let numbers = [45, 56, 1, -4, 6, 7, 8, 9];
+
+function findNumberIndex(number) {
+  for (let i = 0; i < numbers.length; i++) {
+    if (number == numbers[i]) {
+      return i;
+    }
+  }
+  return -1;
+}
+
+function removeElementFromArray(elementToRemove) {
+  for (let i = 0; i <= numbers.length; i++) {
+    if ( numbers[i] === elementToRemove) {
+      numbers.splice(i, 1);
+      console.log(numbers);
+      console.log('findnumberindex ', findNumberIndex(numbers[i]));
+    }
+    }
+  }
+removeElementFromArray(6);
+removeElementFromArray(9);
+removeElementFromArray(45);
+removeElementFromArray(999);
+removeElementFromArray(99);
```

Comment:
Ok, funckja działa, ale polecenie brzmiało: "Prosze użyć funkcji 'findNumberIndex'. Nie uzyła pani tej funkcji, chyba że w console logu, ale to nie jest użycie. Pani Ewo, proszę pamiętac że jeżeli funkcja coś zwraca, można ją traktować jako zmienną. Proszę zrobić tak:
W funkcji 'removeElementFromArray', prosze stworzyć zmienną o jakiejś nazwie i do niej przypisac funkcję 'findNumberIndex' z odpowiednim parametrem. Potem użyć tej zmiennej do wywołania funkcji splice :)